Odysee/Controllers/Library/GoLiveViewController.swift

```diff
@@ -485,8 +511,14 @@ class GoLiveViewController: UIViewController, UIPickerViewDataSource, UIPickerVi
                     return
                 }
 
-                // The claim was successfully set up. Create the stream key and start
-                self.signAndSetupStream(channel: self.channels[0])
+                // The claim was successfully set up.
+                // Wait for the claim to be confirmed, then create the stream key and start
+                let result = data["result"] as? [String: Any]
+                guard let txid = result?["txid"] as? String else {
+                    self.displayRequirementNotMet(message: String.localized("Could not get txid from publish API call."))
```

Comment:
Is this error message okay?

Reply:
We can use a more user-friendly error message: "The livestream could not be created at the moment. Please try again later."

Reply:
What if I put the user friendly message there, but added a `showError` call with this dev-facing message for easier debugging if it's ever needed in future?